docs/docs/database/bca_users_accounts.sql

```diff
@@ -31,6 +32,6 @@ LEFT JOIN contacts c2 on c2.id = cl2.contact_id
 where 
 ps.product_code = 'BCA' 
 and membership_type_code = 'ADMIN'
-and o.id in ('') -- GRAB FROM PAY query.
```

Comment:
You'll probably need to get a list of users from PAY and use them here. ORGS I don't think make sense, because they can be fairly large.